tools/monorepo-utils/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		"rootDir": "src",
 		"outDir": "dist",
-		"target": "es6",
+		"target": "es2018",
```

Comment:
had to bump this target to es2018 as the new typescript check picked up on one of the regex used in this repo being unsupported prior to es2018. According to cursor, es2018 is a safe target since these changes have been supported since node v10